config/initializers/rails_admin.rb

```diff
@@ -76,7 +80,14 @@ def edit_user_link
   end
   config.model('MangaCharacter') { parent Manga }
   config.model('MangaStaff') { parent Manga }
-  config.model('Chapter') { parent Manga }
+  config.model 'Chapter' do
+    parent Manga
+    field :manga_id
```

Comment:
Would this expose `manga_id` as an integer field?  If so, that's probably not the best option 🤔